training/deepspeech_training/util/flags.py

```diff
@@ -93,6 +93,7 @@ def create_flags():
     # Transfer Learning
 
     f.DEFINE_integer('drop_source_layers', 0, 'single integer for how many layers to drop from source model (to drop just output == 1, drop penultimate and output ==2, etc)')
+    f.DEFINE_integer('freeze_source_layers', 0, 'use same value as above to freeze the other layers')
```

Comment:
This description is a bit vague, what is it trying to say?

Reply:
Intended use is together with the 'drop_source_layers' flag, if we want to drop the last layer, we can freeze the rest of the net by setting 'freeze_source_layers' to the same value of 1.

Reply:
Perhaps you could be more explicit in the help text, e.g. say that you have to use the `--drop_source_layers`. Maybe say something like "freeze N remaining layers after using --drop_source_layers"? But even that is a bit unclear. Try and describe what the flag does, and then maybe give an example of how it can be used together with `--drop_source_layers`.

Reply:
Better now?